draft-irtf-panrg-path-properties.md

```diff
@@ -217,7 +219,13 @@ Finally, a NAT that actively modifies IP and TCP/UDP headers based on their cont
 Note that according to this definition, a node that modifies packets in accordance with the hosts, such as a transparent HTTP proxy, as defined in {{RFC2616}}, and a node listening and reacting to implicit or explicit signals, see {{RFC8558}}, are not considered transparent.
 
 Administrative Domain:
-: The administrative domain, e.g., the IGP area, AS, or Service provider network to which a path element belongs.
+: An individual or an organization that owns a path element (or several path elements).
+Examples of administrative domains are an IGP area, an AS, or a service provider network.
```

Comment:
(Responding to the above question, but having to "comment" on two lines to make the suggestion work)

Good point that identifiers for administrative domain are less clearly defined. Perhaps we can leave this as "Administrative domain", and then defining an administrative domain implies providing some kind of identifier or identity that refers to the individual or organization that owns the path element(s) - at least that's the only way I see how one would do that.

If we want to include the word "identity", how about:
```suggestion
: The identity of an individual or an organization that owns a path element (or several path elements).
Examples of administrative domains are an IGP area, an AS, or a service provider network.
```